relayer-cli/src/bin/hermes/main.rs

```diff
@@ -5,28 +5,6 @@
 
 use ibc_relayer_cli::application::APPLICATION;
 
-fn main() -> eyre::Result<()> {
-    install_error_reporter()?;
-
+fn main() {
     abscissa_core::boot(&APPLICATION);
 }
-
-fn install_error_reporter() -> eyre::Result<()> {
-    if !backtrace_enabled() {
-        // If backtraces are disabled, display errors in single line.
-        oneline_eyre::install()
-    } else {
-        // Otherwise, backtraces are enabled, so use the
-        // default error report handler. The impl for
-        // Application::framework_components takes care about
-        // initialization of color_eyre.
-        Ok(())
-    }
```

Comment:
@roman Sorry, this clobbers your work in #1514 because I could not prevent a panic in the new abscissa trying to set up the global eyre handler (which it probably should not do: https://github.com/iqlusioninc/abscissa/issues/589)

BTW, that issue is wrongly linked in the unclog entry as #1515.

Reply:
Reverted this now, but suppressed the color in abscissa terminal components, whatever that does.